library/alloc/src/raw_vec/tests.rs

```diff
@@ -17,17 +18,18 @@ fn allocator_param() {
     // A dumb allocator that consumes a fixed amount of fuel
     // before allocation attempts start failing.
     struct BoundedAlloc {
-        fuel: usize,
+        fuel: RefCell<usize>,
```

Comment:
I think `Cell` should be used here instead.

Reply:
That makes total sense. I will change it.